routes/index.js

```diff
@@ -190,11 +190,73 @@ module.exports = function (passport) {
   });
 
 
-  router.get('/userManagement', checkAuth, function (req, res) {
+  /*router.get('/userManagement', checkAuth, function (req, res) {
     dbAPI.getUsers().then(function (ret) {
       res.render('userManagement.njk', { obj: ret, authenticated: true });
     });
   });
+  */
+
+  router.get('/userManagement', checkAuth, function (req, res) {
+
+    Promise.all([
+      dbAPI.getUsers(undefined),
+      dbAPI.getStations(true)
+
+    ])
+      .then(
+        ([allUsers, allStations]) => {
+          var data = {
+            users: allUsers,
+            stations: allStations,
+            authenticated: true
+          }
+
+          res.render('userManagement.njk', data);
+        }
+      )
+      .catch(
+        err => {
+          console.warn("something went wrong:", err);
+          res.status(500).send(err);
+        }
+      );
+  });
+
```

Comment:
I think you're probably right to leave this function commented out because we're going to edit and make use of this post to search for users again.

Reply:
Now that my DB function is merged I feel like that should be part of this PR. I dont think we should merge it right now until it gets fully implemented.